MODULE.bazel

```diff
@@ -0,0 +1,34 @@
+module(name = "com_github_google_benchmark", version="1.8.0")
+
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "platforms", version = "0.0.6")
+bazel_dep(name = "rules_foreign_cc", version = "0.9.0")
+bazel_dep(name = "rules_cc", version = "0.0.6")
+bazel_dep(name = "rules_python", version = "0.23.1")
+bazel_dep(name = "googletest", version = "1.12.1", repo_name = "com_google_googletest")
+bazel_dep(name = "libpfm", version = "4.11.0")
```

Comment:
this is an optional dependency. is there any way to mark that?

Reply:
I don't think there is a way to express optional dependencies in the new system. But from what I found from my experiments is that the dependency will only be downloaded into the bazel external repo cache when it is actually needed, and that is only the case when the pfm flag is set.

Reply:
ah, i wasn't sure if it was something that would look for a local matching version (installed by the OS package manager) or that it would download.  if it's the latter, then yes that's safer, even if it does download it.